docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# @author Guilherme Baufaker Rêgo (@gbaufake)
+
+# Docker Container Hawkular Metrics Latest Master
+
+FROM maven:3-jdk-8
+
+MAINTAINER Hawkular Metrics <hawkular-dev@lists.jboss.org>
+
+# Install pip
+RUN curl  -o /opt/pip.py "https://bootstrap.pypa.io/get-pip.py"
+RUN python /opt/pip.py
+
+# Install CCM
+RUN pip install ccm
+
+# Start Hawkular Metrics Cassandra Cluster
+CMD ccm create -v 3.0.12 -n 1 -s hawkular
+
+# Clone the repository
+RUN git clone https://github.com/hawkular/hawkular-metrics.git /opt/hawkular-metrics
+
+# http, https, management, debug ports, Cassandra
+EXPOSE 8080 9990 8787 9042
+
+CMD cd /opt/hawkular-metrics/ && mvn clean install -DskipTests
```

Comment:
Shouldn't be RUN instead of CMD? I thought only 1 CMD per dockerfile was allowed.